packages/dbos-cloud/login.ts

```diff
@@ -75,7 +76,14 @@ export async function authenticate(logger: Logger): Promise<string | null> {
   if (!deviceCodeResponse) {
     return null;
   }
-  console.log(`Login URL: ${deviceCodeResponse.verification_uri_complete}`);
+
+  const loginURL = deviceCodeResponse.verification_uri_complete
+  try {
+    await open(loginURL)
```

Comment:
Should we also be adding `open` to `dashboard launch` and `dashboard url`?

Reply:
Good idea--we can add those in the PR that brings back `open`